src/ape_networks/__init__.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+from typing import Dict, List, Optional
+
+from ape import plugins
+from ape.api import PluginConfig
+
+
+class CustomNetwork(PluginConfig):
+    name: str
+    chain_id: Optional[int] = None
+    ecosystem: Optional[str] = None
```

Comment:
What's the behavior when no ecosystem is specified? Should it default to `ethereum` or something?

Reply:
it defaults to the default ecosystem